README.md

```diff
@@ -20,11 +20,11 @@ It is a part of the [OpenMMLab](http://openmmlab.org/) project.
 The master branch works with **PyTorch 1.3+**.
 
 <div align="center">
-  <img src="/docs/imgs/mmaction2_overview.gif" width="450px"/><br>
+  <img src="/resources/mmaction2_overview.gif" width="450px"/><br>
     Action Recognition Results on Kinetics-400
 </div>
 <div align="center">
-  <img src="/docs/imgs/spatio-temporal-det.gif" width="800px"/><br>
+  <img src="/resources/spatio-temporal-det.gif" width="800px"/><br>
```

Comment:
This resource is also used in AVA (under configs) README

Reply:
done